### ISSUE TITLE
some image is 0KB

### ISSUE THREAD

**Issue by ShawnHuang497** [LiangXin1001/LLaVA_Med_qa50_images#2]
hi,
I find some image is 0KB, after i unzipped.
<img width="579" alt="image" src="https://github.com/LiangXin1001/LLaVA_Med_qa50_images/assets/101397888/0875709e-e912-4bf8-b97d-87e400bfd55d">

**Comment by ljpadam** [comment]
Yes, me too. Besides, how do you download these images from PMC? Could you kindly provide the original URLs?

**Comment by ljpadam** [comment]
I use the [final_.jsonl](https://github.com/LiangXin1001/LLaVA_Med_qa50_images/blob/main/final_.jsonl) provided in this repo to download these 50 images, again. Now, every image is normal, no 0 KB.
